test/plugin/scenarios/webflux-scenario/webflux-projectB-scenario/src/main/java/test/apache/skywalking/apm/testcase/sc/webflux/projectB/controller/TestAnnotationController.java

```diff
@@ -44,6 +44,11 @@ public String error() {
         }
         return "1";
     }
+    
+    @RequestMapping("/testcase/webclient/server")
```

Comment:
This should be traced in the expected data. Please recheck.

Reply:
ok,It seems `/testcase/webclient/sercer` finished too fast,I will recheck it.

Reply:
What do you mean `finish too fast`? I can't find the `entry span` and `segment` in your expected data.

Reply:
The span of `/testcaset` finish earlier than `/testcase/webclient/server`,so I'm not sure if this situation is normal.
![image](https://user-images.githubusercontent.com/55177318/94028844-d113c900-fdee-11ea-8fea-c4593be60cc6.png)

From the below picture,we can see the `oap` has received the data which `webclient plugin` sent,but the data of entry span disappeared.So I infer whether `/testcaset` finished too fast so that the data of entry span data was not transfered correctly.
![image](https://user-images.githubusercontent.com/55177318/94028918-e25cd580-fdee-11ea-9824-3975244a5143.png)

Reply:
OK, this screenshot proved the context propagation works, but, we still need in the plugin test, otherwise, we can't be sure this plugin also works.

So, I think the things you mentioned are caused by the complexity of this scenario. It has too many request, so, some parts may be missed when the verification begins.

How about setting up a specific tests for webflux-webclient only?

Reply:
ok，sir，I will try it!

Reply:
@wu-sheng ,Sir,I have write an extra tests for webflux-webclient,but it seems to have a problem during span transmission.
From below picture,we can see that  the exit span named `/projectA/webclient/ping` seems has been sent to oap.But in the second picture ,the entry span seems was not connected with the exit span corresponding with it.So could you please tell me some cause of this result ,or This is because  the `ref data` from `entry span` is not transmitted correctly ?
.![image](https://user-images.githubusercontent.com/55177318/94267567-69848780-ff6e-11ea-94e8-3a371013556f.png)
![image](https://user-images.githubusercontent.com/55177318/94267788-bcf6d580-ff6e-11ea-9f1f-a3b7c154c4f5.png)